components/numeric-value/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { Numeral } from 'design'
+
+const formatNumber = (n, { locale = 'en-GB' } = {}) =>
+  n > 1000
+    ? `${(Math.floor(n / 10) / 100).toLocaleString(locale)}\xa0K`
```

Comment:
Would you mind putting a comment here explaining that magic `\xa0K` meaning because honestly, I don't have any idea?

Reply:
Yes, I will do it when I find it myself 🙂 It's either space or special mathematical char.